src/App.vue

```diff
@@ -12,7 +12,7 @@ export default {
   components: { TheHeader },
   data() {
     return {
-      geolocation: null
+      
     }
     //geolocation should probably stay local 
     //until we determine if multiple components will need it
```

Comment:
Since we have decided to move Geolocation into Vuex's store, we can remove these comments.